src/main/java/uk/ac/ebi/embl/converter/cli/Main.java

```diff
@@ -164,4 +184,61 @@ public static String getFileExtension(Path path) {
         }
         return null; // No extension found
     }
+
+    static class FormatSupportError extends ExitException {
+        public FormatSupportError(final FileFormat fromFt, final FileFormat toFt) {
+            super("Conversion from \"" + fromFt + "\" to \"" + toFt + "\" is not supported");
+        }
+
+        @Override
+        public CLIExitCode exitCode() {
+            return CLIExitCode.UNSUPPORTED_FORMAT_CONVERSION;
+        }
+    }
+}
+
+record CliRulesOption(Map<ValidationRule, RuleSeverity> rules) {}
+
+class RuleConverter implements ITypeConverter<CliRulesOption> {
+    CliRulesOption map = new CliRulesOption(new HashMap<>());
```

Comment:
Since we already have a convert method in this project, it would be easy to follow if we map ValidationRule and RuleSeverity without using ITypeConverter.